oidc.html

```diff
@@ -69,7 +69,7 @@ <h2>Using Lionel Oidc Client</h2>
         clientId: window.config.clientId,
         redirectUri: 'http://localhost:3001/oidc-callback.html',
         postLogoutRedirectUri: 'http://localhost:3001/oidc.html',
-        scopes: ['api'],
+        scopes: ['api', 'email'],
```

Comment:
@dennerkrans yes, but indirectly. Before we merged the id token claims with the access token claims inte the user object.

https://github.com/Aventyret/lionel-oauth-client/pull/110/files#diff-a6527f7775e13a5b2ce3cf51870552a65198115ecfcd7890cb20f5050416b8c2L59

 Since our test provider includes profile info in the access tokens that information was shown in the user object.

Now when we don’t do that the user is empty without any user info related scope.

Reply:
I see. Thanks for clarifying :)